### PR TITLE
feat: display gifs list 🥳

### PR DESCRIPTION
**Pull request by T3kstiil3**
GitHub issue/pull request detail

## Description

List using : https://masonry.desandro.com/
Load with infinite scroll 🥳
List gifs using Giphy api : https://developers.giphy.com/docs/api/endpoint#search

## Screenshot

image

### CODE REVIEW

File: front/src/services/giphy.service.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import axios from 'axios';
+
+const baseUrl = 'https://api.giphy.com/v1/gifs/search';
+const type = 'gifs';
+const apiKey = '3rgXBKNKDmcA5Ykg9i';
```
Comment (ytvnr) [inline front/src/services/giphy.service.js]: sill constrained to have apiKey in the code ?
maybe we can put it in db and encode it ?

File: front/src/services/giphy.service.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+const GiphyService = {
+    search(q, offset = 0, limit = 17) {
```
Comment (ytvnr) [inline front/src/services/giphy.service.js]: why 17? 3x17= 51

File: front/src/views/Home.vue [hunk 3]
```diff
@@ -3,30 +3,26 @@
         <v-container>
             <v-row dense>
                 <v-col cols="12">
-                    <v-card color="green darken-3" dark>
-                        <v-card-title class="headline"
-                            >Welcome,
-                            {{ user.displayName || user.email }}</v-card-title
-                        >
+                    <v-card color="#49868C" dark>
```
Comment (ytvnr) [inline front/src/views/Home.vue]: can you provide screenshot for these new colors please ?

File: front/src/services/giphy.service.js (hunk 2 above)
Comment (T3kstiil3) [reply front/src/services/giphy.service.js]: Because giphy extension use 17 :/

File: front/src/services/giphy.service.js (hunk 1 above)
Comment (T3kstiil3) [reply front/src/services/giphy.service.js]: For the moment is the giphy extension key. Can we do that in a near futur ?

File: front/src/views/Home.vue (hunk 3 above)
Comment (T3kstiil3) [reply front/src/views/Home.vue]: Ups my bad i commit this test :/ 
image

Comment (T3kstiil3) [reply front/src/views/Home.vue]: And i try to put the iframe in a card too

File: front/src/services/giphy.service.js (hunk 1 above)
Comment (ytvnr) [reply front/src/services/giphy.service.js]: for sure !

**Pull request merged**